recipes/sdl/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "2.0.22":
```

If we ever fix the issue, can you temporarily add this to the build matrix so the soon-to-be-released 2.0.24 can be tested.
```suggestion
  "2.0.24-prerelease":  # TEST
      url: "https://www.libsdl.org/tmp/SDL-2.0.zip"
  "2.0.22":
```